apps/smart-contracts/core/test/DepositHook.test.ts

```diff
@@ -128,6 +128,43 @@ describe('=> DepositHook', () => {
       await depositHook.connect(deployer).setDepositRecord(mockDepositRecord.address)
     })
 
+    async function setupDepositPermissionForTest(
```

Comment:
```suggestion
    async function setupDepositPermissionForTest(
```

This was  left behind in the rebase, but is this necessary, given that it is renamed, modified in the next PR?